pkg/eventstore/sql_schema_adapter.go

```diff
@@ -0,0 +1,20 @@
+package eventstore
+
+import (
+	"fmt"
+	"strings"
+)
+
+const eventsTableName = "events"
+
+func defaultInsertMarkers(count int) string {
+	result := strings.Builder{}
+
+	index := 1
+	for i := 0; i < count; i++ {
+		result.WriteString(fmt.Sprintf("($%d,$%d,$%d,$%d,$%d),", index, index+1, index+2, index+3, index+4))
+		index += 5
```

Comment:
`5` is quite magic here, but happens to match the number of columns in both postgreSQL and SQLite adapters

I was thinking about generating some ORM code for the schemas, but I feel that's overkill

supplying `numColumns int = 5` into the function would probably also be trying to make it more generic than it needs to.

So let's just stay with what we have 😓

Reply:
done: https://github.com/ThreeDotsLabs/esja/pull/8/commits/0dc221042c2f55e6b7a9ad08e34a40ae00cc6292

Reply:
thanks for that catch, @maclav3